_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -18,6 +18,10 @@ You can configure the `add_entries` processor with the following options.
 | :--- | :--- | :--- |
 | `entries` | Yes | A list of entries to add to an event. |
 | `key` | Yes | The key of the new entry to be added. Some examples of keys include `my_key`, `myKey`, and `object/sub_Key`. |
+| `metadata_key` | Yes | The key for the new metadata attribute. Argument must be a literal string key and not a JsonPointer. Either one string key or `metadata_key` is required. |
+| `format` | No | A format string to use as the value of the new entry, for example, `${key1}-${key2}` where `key1` and `key2` are existing keys in the event. Required if `value` and `value_expression` is not specified. |
+| `value_expression` | No | An expression string to use as the value of the new entry, for example, `/key` is an existing key in the event with a type of either a number, a string, or a Boolean. Expressions can also contain functions returning Number/String/Integer. For example, `length(/key)` would return the length of the key in the event when the key is a string. For more information about keys, see [Expression syntax](https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/). Required if `value` and `format` are not specified. |
```

Comment:
Same comment: "Required if neither `value` nor `format` is specified" or "Required if both `value` and `format` are not specified"? Also, is "Number/String/Integer" intentionally capitalized, or should it be lowercase?